Connector Documentation - Adyen and Banked (#27)
gr4vy/gr4vy-docs: docs/connectors/banked-banked.mdx

---
title: Banked
sidebar_label: Banked
slug: /connectors/banked-banked
---
import Image from "@site/src/modules/Image";
import api_key_image from "./assets/banked_api_key.png";
import merchant_account_image from "./assets/adyen_merchant_account.png";

## Sign-up
Banked provides a self-service sign-up for a sandbox account.  To sign-up for an account simply visit the [sign-up page](https://console.banked.com/users/sign_up) and fill in the details.

## Credentials

> * **Client ID**

Reviewer: ```suggestion
### Client ID
```

  The Banked Client ID is obtained from the Banked Admin Portal under the `Applications` -> `<Your Application>` -> `Configuration` tab.

>   <Image src={api_key_image} title="API Key" />

Reviewer: Stupid question maybe: Would this not have worked as a simple markdown file and an image? 

E.g. `[Image](./assets/banked_api_key.png)`?

> * **Client secret**

Reviewer: ```suggestion
### Client secret
```

>   The Banked Client secret is obtained from the same tab as the Client ID.  See above.

Reviewer: ```suggestion
  The Banked Client secret is obtained from the same tab as the [Client ID](#client-id).
```

This is now possible cause Client ID is now a header

> * **Payee name**

Reviewer: ```suggestion
### Payee name
```

  The name of the Payee, i.e. the person or company the payment is to be made to.  

  

>   :::info Payee Details
>   
>   NOTE: By default you are only permitted to create payment sessions with payee details that match an account linked to the console. In test you will have a fake bank account created automatically when signing up to the console. Unless you connect additional sandbox accounts, you must use the following payee details:
> 
>   * account_number 12345678
>   * sort_code 010203
>   
>   :::

Reviewer: ```suggestion
  :::caution Payee Details
  
  By default you are only permitted to create payment sessions with payee details that match an account linked to the console. In test you will have a fake bank account created automatically when signing up to the console. Unless you connect additional sandbox accounts, you must use the following payee details:

  * account_number 12345678
  * sort_code 010203
  
  :::
```

Rather than using NOTE in the string we can make the block orange by using a `caution` admonition. https://docusaurus.io/docs/next/markdown-features/admonitions

> * **Payee account number**

Reviewer: ```suggestion
### Payee account number
```

>   The account number of the Payee, i.e. the person or company the payment is to be made to. See Payee name for more details.

Reviewer: ```suggestion
  The account number of the Payee, i.e. the person or company the payment is to be made to. See [Payee name](#payee-name) for more details.
```

> * **Payee sort code**

Reviewer: ```suggestion
### Payee sort code
```
